### DISCUSSION TITLE
Why to use `eval` here?

### DISCUSSION THREAD

**Discussion by olorin37** [LunarVim/LunarVim#2339] [Q&A] [answered]
https://github.com/LunarVim/LunarVim/blob/637658ae2c4ae6acd2179ea5ce7a482e557944f2/utils/installer/install.sh#L267

Just invoking the string is enough.

like this

```bash
    "__install_nodejs_deps_$pkg_manager" 
```

**Comment by kylo252** [comment, marked as answer]
That's correct! I assume it may have been done at the time when this was a command instead of a function call, that's where `eval` would be necessary.

**Reply by olorin37** [reply]
Actually it is no a difference between ivoking function or command in bash in this subject.
